languages/scala/recursive_descent/Parser_scala_recursive_descent.ml

```diff
@@ -298,15 +274,18 @@ let adjustSepRegions lastToken in_ =
         else
           (* stricter: original code just does nothing *)
           error "unmatched closing token" in_
-    (* pad: not that an arrow can also be used outside of a case, so
-     * here we dont raise an error if we don't find a match.
-     *)
-    | (ARROW _ as x), y :: ys when x =~= y -> ys
     | _, xs -> xs
   in
   in_.sepRegions <- newRegions;
   ()
 
+let inSepRegion tok f in_ =
```

Comment:
this code was in a different PR too ...

Reply:
same as before, i think it's because of the PRs being merged out of order (i was planning on merging them myself after approval)